remote/theme.json

```diff
@@ -20,7 +20,7 @@
 			"palette": [
 				{
 					"slug": "background",
-					"color": "#222222",
+					"color": "#000000",
```

Comment:
Looking at the figma files, I see #222 is what's being used but the site does look lighter than the designs. @kjellr can you give us your expert eye?

Reply:
`#000000` is the correct background color here.